src/pages/myNotes/note/note.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@material-ui/core';
+import useStyles from '../styled';
+
+const Note = ({ title, description, date, changeState, id, notActive, changeText }) => {
+  const classes = useStyles();
+  const cutSymbols = () => {
+    let str = '';
+    if (description.length > 20) {
```

Comment:
I think we can remove this check:
`description.length > MAX_DESC_LENGTH ? '${description.substring(0, MAX_DESC_LENGTH)}...' : description`